ConsumerAPI/NodeJs/main.js

```diff
@@ -7,11 +7,10 @@ const identity = {
     token: 'ZHVtbXlfdG9rZW4=',
 };
 
-const PROCESS_MODEL_ID= 'Lager-Manuell';
-//const PROCESS_MODEL_ID= 'Lager-Teilautomatisch';
+const PROCESS_MODEL_ID= 'Benutzeraktivierung';
```

Comment:
```suggestion
const processModelId = 'Benutzeraktivierung';
```

In der .ts Version schreiben wir auch Konstanten auf diese Weise.

(Wäre auch vllt. eine Überlegung wert ein Typescript Sample bereitzustellen?)